### PR TITLE
Add to google calendar

### PR DESCRIPTION
**Pull request by TimMurphy**
Add calendar option 'Add to Google Calendar'
- Update home page model.
- Add calendar option on home page
- Update deploy.cmd to use C# 6.0 compatible compiler
#26

### CODE REVIEW

File: deploy.cmd [hunk 1]
```diff
@@ -57,9 +57,9 @@ IF DEFINED CLEAN_LOCAL_DEPLOYMENT_TEMP (
   mkdir "%DEPLOYMENT_TEMP%"
 )
 
-IF NOT DEFINED MSBUILD_PATH (
-  SET MSBUILD_PATH=%WINDIR%\Microsoft.NET\Framework\v4.0.30319\msbuild.exe
-)
+:: Explicitly override MSBUILD_PATH to use C# 6.0 compatible compiler.
+SET MSBUILD_PATH=%ProgramFiles(x86)%\MSBuild\14.0\Bin\MSBuild.exe
```
Comment (davidfowl) [inline deploy.cmd]: This actually doesn't matter because we don't use msbuild to compile anything.

Comment (TimMurphy) [inline deploy.cmd]: Agreed. Should have looked further down deploy script :-)

File: src/live.asp.net/ViewModels/HomeViewModel.cs [hunk 2]
```diff
@@ -28,5 +28,21 @@ public class HomeViewModel
...
 
         public bool ShowMoreShowsUrl => !string.IsNullOrEmpty(MoreShowsUrl);
+
+        public string AddToGoogleUrl
+        {
+            get
+            {
+                // reference: http://stackoverflow.com/a/21653600/22941
+
+                const string eventTitle = "ASP.NET Community Standup";
```
Comment (DamianEdwards) [inline src/live.asp.net/ViewModels/HomeViewModel.cs]: We only use `const` for actual constant things that will never ever change. Granted, in an application it doesn't make much difference as nothing will be referencing it, but I'd like to keep the coding style consistent.

File: deploy.cmd (hunk 1 above)
Comment (DamianEdwards) [inline deploy.cmd]: Can you remove that change please?

**Comment by TimMurphy** [comment]
@DamianEdwards completed your 2 requests.

I'm a n00b to pull requests & code reviews. Is it best practice for me to 'Add a line note' to each request when I've push the commit?

File: src/live.asp.net/ViewModels/HomeViewModel.cs [hunk 3]
```diff
@@ -9,6 +9,8 @@ namespace live.asp.net.ViewModels
 {
     public class HomeViewModel
     {
+        private static string _dateTimeFormat = "yyyyMMddTHHmmssZ";
```
Comment (DamianEdwards) [inline src/live.asp.net/ViewModels/HomeViewModel.cs]: Make it `public static readonly string`

**Comment by DamianEdwards** [comment]
@TimMurphy always a good idea to include a descriptive commit message

File: src/live.asp.net/ViewModels/HomeViewModel.cs (hunk 3 above)
Comment (TimMurphy) [inline src/live.asp.net/ViewModels/HomeViewModel.cs]: Agree it should be `readonly` but why `public`? The value isn't used, nor intended to be used, outside of the view model.

I followed your lead from [iCalendarOutputFormatter.cs](https://github.com/aspnet/live.asp.net/blob/dev/src/live.asp.net/Formatters/iCalendarOutputFormatter.cs). Should I refactor? To say DateTimeExtensions.ToIso8601()?

Comment (DamianEdwards) [inline src/live.asp.net/ViewModels/HomeViewModel.cs]: My bad, I meant private.

**Pull request merged**